rules.tf

```diff
@@ -576,11 +596,29 @@ resource "aws_wafv2_web_acl" "default" {
           content {
             arn = rule_group_reference_statement.value.arn
 
-            dynamic "excluded_rule" {
-              for_each = lookup(rule_group_reference_statement.value, "excluded_rule", null) != null ? toset(rule_group_reference_statement.value.excluded_rule) : []
+            dynamic "rule_action_override" {
```

Comment:
leave `excluded_rule` as is to prevent breaking change, and instead add new option for `rule_action_override`